test/scripts/start-services.sh

```diff
@@ -62,32 +88,15 @@ start_parachain()
 
     echo "Writing Polkadot configuration"
     cp config.json $configdir/polkadotConfig.json
-    parachain_conf="{
-        \"bin\": \"$bin\",
-        \"id\": \"200\",
-        \"wsPort\": 11144,
-        \"port\": 31200,
-        \"flags\": [
-            \"--execution=native\",
-            \"-lruntime=debug,import_header=trace,bridge=trace\",
-            \"--rpc-cors=all\",
-            \"--offchain-worker=Always\",
-            \"--enable-offchain-indexing=true\",
-            \"--\",
-            \"--execution=wasm\"
-        ],
-        \"chain\": \"$configdir/spec.json\"
-    }"
     node ../test/scripts/helpers/overrideParachainSpec.js $configdir/polkadotConfig.json \
-        parachains.0 "$parachain_conf"
+        parachains.0.bin "$bin" \
+        parachains.0.flags.0 "--execution=native"
```

Comment:
Hmm.... so I would prefer the test services explicitly stating the ports and log files they use here ("id" field is used as log name). Otherwise these details are scattered all over the codebase. It's easy to break this startup script by changing a port in what looks like an entirely independent project

Reply:
well i suppose now they just use whatever is default in config.json - this file is only really used for e2e tests now too, isn't it? so if we move it to this test folder, then that would be explicit?

Reply:
though i'm not sure - is this used for anything else?

Reply:
Agree with Riz, the config in `parachain/config.json` has other uses besides the E2E tests (like for testing XCMP, which isn't yet testable as part of the E2E tests).

Reply:
fixed